src/main/java/core/basesyntax/ObtainArea.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax;
+
+public interface ObtainArea {

```

Comment:
```suggestion
public interface AreaCalculator {

```

погана практика починати назви класів з дієслів